src/__tests__/ProductView/EventTable.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import EventTable from '../../ProductView/EventTable/EventTable.js';
+import renderer from 'react-test-renderer';
+import TestEventTable from './../testData/eventTable.js';
+let instance;
+
```

Comment:
Maybe it would be reasonable to also test the functions testFilterArguments and testFilter explicitly in order to cover edge cases or get a more sufficient feedback in case of an error.
What do you think?